src/app/components/content-page/content-page-variables.scss

```diff
@@ -0,0 +1,20 @@
+/* header , nav, main, bottom bar ,content*/
```

Comment:
These are okay for now - but in production code they need to be self-explanatory and notes like this will go into the commit message.

Reply:
I remove the comments!